maproulette/src/main/java/com/pixeltron/mapquest/open/geocoding/RequestParameters.java

```diff
@@ -0,0 +1,29 @@
+package com.pixeltron.mapquest.open.geocoding;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class RequestParameters {
```

Comment:
This is a great change! Makes the parent code much cleaner.

An option could be to make this an interface, and have separate implementations for geocode vs foursquare (similar to the `Request` object). This would make it more consistent from a design pattern (eg I need FoursquareRequestParameters to make a FoursquareRequest) and would allow for having the class only set the necessary parameters.

Additionally, depending on how open or closed you intend this class to be, it might be useful to make these variables `final`. This would prevent issues where certain parameters may get changed unintentionally between uses of the instance.